server/controller/properties.js

```diff
@@ -0,0 +1,11 @@
+const db = require('../database');
+
+const properties = (request, response) => {
+  const { propertyId } = request.params;
+  db.query(`SELECT * FROM properties WHERE propertyId = ${propertyId}`)
+    .then(data => {
+      response.send(data.rows[0]);
+    });
+}
```

Comment:
Are we supposed to be sending http codes back here with the data?

Reply:
Will fix this